plugins/woocommerce-admin/client/marketplace/components/product-list-content/no-results.tsx

```diff
@@ -90,6 +90,8 @@ export default function NoResults(): JSX.Element {
 					className="woocommerce-marketplace__no-results__icon"
 					src={ NoResultsIcon }
 					alt={ __( 'No results.', 'woocommerce' ) }
+					width="80"
+					height="80"
```

Comment:
This is set to 100px in `no-results.scss:16`, not sure if that means it should be 100px here too.

Reply:
Oops! Design calls for 80px; it's the SCSS that's wrong! Corrected in 98ac373b1bc6adb352304b798300a2a058e1e8a3.

(I've only added it here as well to allow the browser to reserve approximately the right amount of space for the image before it or the CSS loads, reducing CLS.)